docs/detections/alerts-add-to-cases.asciidoc

```diff
@@ -7,7 +7,7 @@
 
 From the Alerts table, you can attach one or more alerts to a <<signals-to-new-cases,new case>> or <<signals-to-existing-cases,an existing one>>. Alerts from any rule type can be added to a case.
 
-NOTE: Once you've added an alert to a case, you can only remove it through the <<cases-api-overview,Elastic Security Cases API>>.
+NOTE: After you add an alert to a case, you can remove it from the case history or by using the <<cases-api-overview,Elastic Security Cases API>>.
```

Comment:
> ... you can remove it from the case history ...

I'm not familiar with the term "case history". It doesn't seem to be mentioned in https://www.elastic.co/guide/en/security/current/cases-open-manage.html#cases-examine-alerts either. Should we perhaps be clearer about where to find that functionality?

Reply:
Sorry, just saw this now @lcawl. Looking back, I don't think we ever formally defined the area under the case summary. Internally, I think we've referred to it as the "case history" or "case activity", and it wasn't until `8.4` that we visually identified it in the UI as the [Activity](https://www.elastic.co/guide/en/security/8.3/cases-open-manage.html#cases-summary) tab (see example [here](https://www.elastic.co/guide/en/security/8.3/cases-open-manage.html#cases-summary)).

Wdyt about changing "case history" to "case activity under the alert summary" or "case activity thread"?

Reply:
Ah yes, thanks, I think referring to it by the name that matches the UI is clearer.

Reply:
```suggestion
NOTE: After you add an alert to a case, you can remove it from the case activity under the alert summary or by using the <<cases-api-overview,Elastic Security Cases API>>.
```